blazar/utils/openstack/placement.py

```diff
@@ -467,6 +467,32 @@ def _get_custom_reservation_trait_name(self, reserv_uuid, project_id):
         project_id = project_id.upper().replace("-", "_")
         return "CUSTOM_RESERVATION_" + reserv_uuid + "_PROJECT_" + project_id
 
+    def is_trait_exist(self, trait_name):
```

Comment:
This looks like it should probably be a "private" function, as it's general purpose and I don't think callers of this client should be using it directly

```python
def _trait_exists(self, trait_name):
    # ...
```